plugins/ua_config_default.c

```diff
@@ -807,9 +807,11 @@ UA_ClientConfig_setDefaultEncryption(UA_ClientConfig *config,
                        UA_StatusCode_name(retval));
     }
 
-    if(config->securityPoliciesSize == 0) {
+    // Policy "None" is already present before the encryption related security policies are added
```

Comment:
Failing to load a certificate should not lead to the removal of policy-none.
This is only required as a malloc cleanup when there is no element in the array.

Reply:
As a user of the API, I would not expect a function named "UA_ClientConfig_setDefaultEncryption()" to give me a client config which can only do the "None" security policy.

But as it would be better not to change behavior from one 1.1 release to the next, I'll restore the old code for this part.